src/commands/source/download.rs

```diff
@@ -314,5 +314,9 @@ pub async fn download(
     }
 
     debug!("r = {:?}", r);
+
+    let verify_result = super::verify(matches, config, repo, progressbars).await;
+    debug!("verify_result = {:?}", verify_result);
```

Comment:
Does this do anything? Looks like it only adds debug output (assuming `verify` hopefully doesn't panic).

Reply:
Fixed now and I also forgot to consider that `super::verify` can emit output (so it already was partially useful).